roadmap/6_infrastructure/in-progress/2_Remuneration_V2.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Remuneration V2
+links:
+  Forum Post: https://forum.dfinity.org/t/the-state-and-direction-of-decentralization-nodes-on-the-internet-computer/9170/178
```

Comment:
```suggestion
  Forum Post: https://forum.dfinity.org/t/the-state-and-direction-of-decentralization-nodes-on-the-internet-computer/9170
```
Link to thread instead of specific post